news/2025/2025-03-05-mappers-guild-spring-2025-priority-quests.md

```diff
@@ -0,0 +1,89 @@
+
+The Mappers' Guild is open to mappers of any skill level, including those without Ranked maps! See [the system's website](https://mappersguild.com/) or the [osu! Discord server](https://discord.com/invite/ppy)'s `#mappers-guild` channel for details!
+
+## What are priority quests?
+
+- **Anyone can participate in [priority quests](https://mappersguild.com/missions).** Unlike [normal quests](https://mappersguild.com/quests), you don't need to claim priority quests in advance.
+- **Quests are tiered by difficulty:**
+  - Tier 1 ![bronze icon](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/bronze-small.png) (easiest, lowest reward)
+  - Tier 2 ![silver icon](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/silver-small.png)
+  - Tier 3 ![gold icon](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/gold-small.png)
+  - Tier 4 ![platinum icon](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/platinum-small.png) (hardest, highest reward)
+- **Multiple users can complete the same quest**. Most of the time, anyone who accomplishes the quest's objective before the deadline is eligible.
+
+By completing priority quests, you can earn some special rewards too:
+
+- All the usual [Mappers' Guild rewards](/wiki/Community/Mappers_Guild#rewards) for ranking beatmaps
+- A [**customised keychain**](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/keychain.jpg) based on a pattern in your map
+- A profile badge upon completing 3 priority quests
+
+![Priority quest badge](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/questbadge.png)
+
+## New priority quests
+
+**6** new quests have just been added to the [priority quest listing](https://mappersguild.com/missions), all of which will be open to mappers until **May 15, 2025**!
+
+Below is an overview of each quest. If any are confusing, feel free to ask questions in the comments or directly to [pishifat](https://osu.ppy.sh/community/chat?sendto=3178418). Be sure to check the quest pages for additional requirements!
+
+### [Point of entry](https://mappersguild.com/missions?id=67c3b7f965c8e39f49e79240) ![bronze icon](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/bronze-small.png)
+
+When installing osu! for the first time, your client can automatically fill your library with a few maps. We call these "bundled maps", which are randomly selected from a pool of curated Featured Artist maps.
+
+**Your goal is to create a map eligible for the bundled map pool!** This has a lot of weirdly specific requirements detailed [here](https://github.com/bdach/osu/blob/master/osu.Game/Beatmaps/Drawables/BundledBeatmapDownloader.cs#L134-L152).
```

Comment:
seems to be the first time that mappers can map directly into the bundled listing 🤔
will every eligible map be bundled or only a small curated subset?

Reply:
there's a difference between what contest winners get (consistent bundling in every install) vs the rotating bundled map pool (hundreds of maps are here already, and eligible ones from this quest can be added here, so long as they fit requirements + my discretion for weird cases)